frontend/package.json

```diff
@@ -51,16 +54,18 @@
     "import-glob-loader": "^1.1.0",
     "jest": "^24.9.0",
     "node-sass": "^4.13.0",
+    "prettier": "2.0.2",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-hot-loader": "^4.12.16",
```

Comment:
Don't these belong in `devDependencies`?

Reply:
Package has been cleaned up now, so should mostly match stash.